data/users.yml

```diff
@@ -1053,6 +1053,9 @@
   image: productive.png
   alt: Productive
   use: "Productive uses Ember for its main web applications."
+  - url: "https://www.fastly.com/"
+  alt: Fastly
```

Comment:
@amyrlam could you tweak the image name to just `fastly.png` to match the other logos and then add a line here that says `image: fastly.png` ? Without that line, I don't think this will show up ...

Reply:
Fixed! Sorry, copied it wrong. Also my fork was out of date, resolved now!